src/styles/scss/shared.scss

```diff
@@ -43,6 +43,13 @@
   transition: 0.3s;
   z-index: 2;
 
+  &.modal-btn-close--offset {
+    @include breakpoint-s {
+      top: 100px;
+      right: 100px;
+    }
+  }
+
```

Comment:
Consider moving all the modal styling from `shared.scss` to `modal.scss`.

I do not see a reason why it should be part of a generic file when we have a file that deals with this area of the system.

If you do this please also include `.modal-btn-fallback` further down this file. It is outside the scope of this PR though.